user/caribou/module/src/H2MEventConverter.cc

```diff
@@ -119,8 +135,34 @@ bool H2MEvent2StdEventConverter::Converting(
     uint64_t timestamp = pixHit->GetMode() == caribou::ACQ_MODE_TOA ? (frameEnd - (pixHit->GetToA() * _100MHz_to_ps)) : not_toa_time;
     uint64_t tot = pixHit->GetToT();
 
+    // best guess for charge is ToT if no calibration is available
+    EUDAQ_DEBUG("tot is: " + to_string(tot));
+
+    double charge = static_cast<float>(tot);
+    if(!vtot.empty()) {
+        EUDAQ_DEBUG("Applying calibration to DUT");
+        size_t scol = static_cast<size_t>(col);
+        size_t srow = static_cast<size_t>(row);
+        float a = vtot.at(64 * srow + scol).at(2);
```

Comment:
what's stored in `vtot.at(X).at(0)` and `vtot.at(X).at(1)`? If it's just the coordinates, I would drop them when reading the file since you don't need them